### ISSUE TITLE
Custom CSS

### ISSUE THREAD

**Issue by camaech** [avil13/vue-sweetalert2#50]
I'd like to use a custom theme for sweetalert2, but you are importing sweetalert2.min.css straight from the sweetalert2 package.

Anyway you could add a parameter such as this:
`Vue.use(VueSweetalert2, {includeCss: false});`

That way I could use webpack to compile sweetalert2 from source and override any variables that I want.

**Comment by avil13** [comment]
@camaech I tried to add the parameter you are asking for.
But webpack loads styles even if you don’t ask them.
If you have any idea how to solve this, I will say thank you.

**Comment by camaech** [comment]
@avil13 I think it should be in your index.js file where you need to add an if/else.

`/** @type swal {import("sweetalert2")} */
import swal from 'sweetalert2/dist/sweetalert2.min.js';
import 'sweetalert2/dist/sweetalert2.min.css';`

**Comment by avil13** [comment]
Sorry. In this branch.
It has not yet been added to the master branch.

https://github.com/avil13/vue-sweetalert2/tree/feature/v2

**Comment by avil13** [comment]
You can try this so `npm i vue-sweetalert2@beta`

**Comment by camaech** [comment]
I can't get this working. My project won't compile now.

**Comment by avil13** [comment]
@camaech Can you show error messages?

**Comment by camaech** [comment]
94% after seal

 ERROR  Failed to compile with 1 errors                                                                                                                                                                   11:46:51 AM

This dependency was not found:

* vue-sweetalert2 in ./resources/js/testing/app.js

To install it, you can run: npm install --save vue-sweetalert2
                   Asset      Size  Chunks             Chunk Names
   /css/flickity.min.css  1.79 KiB          [emitted]
/js/flickity.pkgd.min.js  52.5 KiB          [emitted]
 + 4 hidden assets

**Comment by camaech** [comment]
@avil13 All I'm doing is this:

`import VueSweetalert2 from 'vue-sweetalert2'`

**Comment by avil13** [comment]
Is my fail. Need to add prepublish compiling

**Comment by camaech** [comment]
@avil13 Let me know when I can try it again.

**Comment by avil13** [comment]
@camaech 
Updated. Try again `npm i vue-sweetalert2@beta`

**Comment by camaech** [comment]
@avil13 I think I fixed this in https://github.com/avil13/vue-sweetalert2/pull/51. Please review.

**Comment by avil13** [comment]
Works, thank you. Now I will correct the README and publish.

**Comment by camaech** [comment]
Here's an example of the scss file you can include to override the base sweetalert theme variables:


```
@import '~sweetalert2/src/variables';

$swal2-background: #990000;

@import '~sweetalert2/src/sweetalert2';
```

This is assuming that you set your includeCss = false

**Comment by avil13** [comment]
https://github.com/avil13/vue-sweetalert2/blob/feature/v2/README.md#additional-options

**Comment by camaech** [comment]
Looks great. Go ahead and close this issue.

**Comment by marcbelletre** [comment]
It looks like this is not working anymore.

I included the JS only in app.js
```js
Vue.use(VueSweetalert2)
```

And I included the CSS in my app.scss file
```scss
// [...]
$swal2-confirm-button-background-color: $primary;

@import '~sweetalert2/src/sweetalert2';
```

The problem is that the Sweetalert renders the CSS in a style tag which overrides every style from my main CSS file. 

![Screenshot 2020-04-09 at 18 55 33](https://user-images.githubusercontent.com/6853334/78920542-d9bc5100-7a93-11ea-89b4-4496f995eea2.png)
![Screenshot 2020-04-09 at 18 56 07](https://user-images.githubusercontent.com/6853334/78920538-d88b2400-7a93-11ea-8485-3e9b54aaf493.png)

If I just remove the style tag from the web inspector I can actually see my custom styles:

![Screenshot 2020-04-09 at 18 56 17](https://user-images.githubusercontent.com/6853334/78920537-d7f28d80-7a93-11ea-8933-a3697c1c22e8.png)

Is there a way to prevent Sweetalert from creating the style tags? I tried using `includeCss: false` in the options but it didn't work. Also it is not documented.

**Comment by avil13** [comment]
Hi. 
This is the problem with adding classes in sweatalert2. 

Can add issue there.
https://github.com/sweetalert2/sweetalert2

Or override the value of the styles for example:
```css
#body .swal2-popup.swal2-toast {
 /* Your styles */
}
```

```html
<body id="body">
...
</body>

```

**Comment by marcbelletre** [comment]
Hi @avil13 

Thank you for your help.
Actually I think the issue comes from this package. After further research I just found out that the problem is solved if you just replace the first import in the `dist/index.js` file to:
```js
import Swal from 'sweetalert2/dist/sweetalert2.js';
```

Instead of:
```js
import Swal from 'sweetalert2';
```

This was already fixed by #51 but it looks like it has since been reverted.
